pkg/api/orchestrators.go

```diff
@@ -197,7 +197,7 @@ func kubernetesUpgrades(csOrch *OrchestratorProfile) ([]*OrchestratorProfile, er
 			continue
 		}
 		// add patch upgrade
-		if nextVersion.Major() == currentMajor && nextVersion.Minor() == currentMinor && currentPatch < nextVersion.Patch() {
+		if nextVersion.Major() == currentMajor && nextVersion.Minor() == currentMinor && currentVer.LessThan(nextVersion) {
```

Comment:
@dmitsh This change (and the lines above it) allow the existing upgrade whitelist logic to work for "pre-release" versions, e.g.: `v1.10.0-beta.2` --> `v1.10.0-beta.4`